app/models/backend_api.rb

```diff
@@ -37,25 +37,20 @@ class BackendApi < ApplicationRecord
 
   has_system_name(uniqueness_scope: [:account_id])
 
-  scope :orphans, -> { where.has { id.not_in(BackendApiConfig.selecting { :backend_api_id }) } }
+  scope :orphans, -> {
+    where.has do
+      not_exists(BackendApiConfig.except(:order).by_backend_api(BabySqueel[:backend_apis].id).select(:id))
+    end
+  }
```

Comment:
What generates for Oracle now for `BackendApi.orphans.explain`:

```sql
SELECT "BACKEND_APIS".*
FROM "BACKEND_APIS"
WHERE (
  NOT EXISTS(
    SELECT "BACKEND_API_CONFIGS"."ID"
    FROM "BACKEND_API_CONFIGS"
    WHERE "BACKEND_API_CONFIGS"."BACKEND_API_ID" = "BACKEND_APIS"."ID"
  )
)
```

![image](https://user-images.githubusercontent.com/11318903/71095984-d54eee00-21ad-11ea-9ab3-a784a28c3b00.png)